pkg/minikube/cruntime/containerd_test.go

```diff
@@ -27,7 +27,7 @@ func TestAddRepoTagToImageName(t *testing.T) {
 		imgName string
 		want    string
 	}{
-		{"kubernetesui/dashboard:v2.1.0", "docker.io/kubernetesui/dashboard:v2.1.0"},
+		{"kubernetesui/dashboard:v2.5.0", "docker.io/kubernetesui/dashboard:v2.5.0"},
 		{"kubernetesui/metrics-scraper:v1.0.4", "docker.io/kubernetesui/metrics-scraper:v1.0.4"},
```

Comment:
In the previous upgrade, this file has not been updated, so metrics-scraper is not the latest version.
I think it's better to update from v1.0.4 to v1.0.7.

Reply:
It was corrected and thank you so much for letting me know!